### PR TITLE
adding nextflow files

### PR DESCRIPTION
**Pull request by tariship**

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+trace {
+    enabled = true
+    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
+}
+dag {
+    enabled = true
+    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.html"
+}
+
+manifest {
```
Comment (nvictus) [inline nextflow.config:166]: Does this need updating?

File: docker/mtx.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
+// The mtx.json schema is documented on Confluence here:
```
Comment (nvictus) [inline docker/mtx.json]: potentially sensitive file?

**Review by nvictus — APPROVED** [review]
LGTM. Flagged a couple files just in case there is sensitive info there. Otherwise good to merge.

**Comment by nvictus** [comment]
Oops. Was that an accident?

**Comment by tariship** [comment]
Yup. Just forked it again and will make a new pull request in a bit. Sorry about that!

**Pull request closed without merging**